### ISSUE TITLE
followed all the steps but still no OSD

### ISSUE THREAD

**Issue by bencapturesdotcom** [fpv-wtf/wtfos-configurator#252]
ive watched a few different peoples videos to see what i missed and i cant work it out. everything seemed to go smoothly. and when i power everything up i can see new text in the bottom right but all i get is "osd waiting..." see pic

has anyone got any ideas?
![IMG_4915](https://user-images.githubusercontent.com/120987530/208317785-3d92ca52-ec7c-4594-88eb-81feffcda070.jpeg)

**Comment by stylesuxx** [comment]
OK, so on the goggle side everything seems to be fine, so this leaves the air unit. Most likely you don't have MSP enabled on the correct port - show us your BF config.

The steps to enable within BF are all outlined here: https://github.com/fpv-wtf/msp-osd
Double check that you did all of that.

**Comment by bencapturesdotcom** [comment]
thanks for the speedy reply! yes ive done all that and I first used the BF CLI to determine the UART, ive now just tried your preset and im getting the same results. what screen of BF config would you like me to screenshot?

**Comment by stylesuxx** [comment]
Show the port configuration and post a dump of your config. Also - do you see the native OSD? For some reason I can't see your attached image

**Comment by djgoldeneye** [comment]
Hello having similar issue. Im getting the DJI Styled OSD elements but cant get the betaflight version to load. It was working before then stopped. I tried my other quad with that hasnt been updated for HD OSD and it works everytime. Cant get this one to work on Betaflight 4.3 or the HD version on 4.4.

**Comment by stylesuxx** [comment]
@djgoldeneye please open a separate issue. If it has been working on your end before we probably have a different issue here.

You can connect to wtfOS configurator still on the air unit where it stopped working? Chances are that your issue is that slots changed, the configurator will tell you so and will probably fix it.

**Comment by bencapturesdotcom** [comment]
> 

thanks for the help. ive been reading around on here, is this something better solved on discord?

hopefully these screenshots work this time.

<img width="1440" alt="Screenshot 2022-12-19 at 16 44 07" src="https://user-images.githubusercontent.com/120987530/208476945-8aab97bb-9be0-4ba4-9292-4b14333bb845.png">
<img width="1440" alt="Screenshot 2022-12-19 at 16 43 46" src="https://user-images.githubusercontent.com/120987530/208476951-89057cde-36b6-459a-a5e0-f9b038ff61cc.png">
<img width="1440" alt="Screenshot 2022-12-19 at 16 43 23" src="https://user-images.githubusercontent.com/120987530/208476954-62a8e321-10da-48cf-b9cd-1dcd01933cbf.png">

**Comment by stylesuxx** [comment]
Every support request is probably solved easier on discord because you have hundreds of people there instead of max 5 who check here ;-)

Anyway, I am still missing the output of `dump` in the cli.

**Comment by djgoldeneye** [comment]
> @djgoldeneye please open a separate issue. If it has been working on your end before we probably have a different issue here.
> 
> You can connect to wtfOS configurator still on the air unit where it stopped working? Chances are that your issue is that slots changed, the configurator will tell you so and will probably fix it.

I was able to get it working again. Here's what I noticed. When attempted to remove and reinstall WTFOS, the package manager would be empty. I would still try to select all and search for dji osd but still wouldn't work. 

Not until I was able to install WTFOS and be able to see the 8 or so packages without filtering was able to see the osd. 
I had to use a macbook instead of a pc to get it working.

**Comment by bencapturesdotcom** [comment]
im a n00b to most things fpv related, github and discord included. thanks for your time. some people on discord mentioned BF 4.4 which is what ive flashed to my quads and the osd is all working now with that new drop down menu in the peripherals.
